lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -7759,7 +7759,6 @@ func TestMakeRemapDotConfigMidNoCacheRemapLineTopo(t *testing.T) {
 
 	dsType := tc.DSType("HTTP_NO_CACHE")
 
-	// BNO
```

Comment:
Random deletion, but looks like it needs to be.

Reply:
Unrelated cleanup while scanning for some debug things I throw in there.